addon/components/hyper-table-v2/column.hbs

```diff
@@ -1,10 +1,37 @@
-<div class={{this.computedClasses}} ...attributes>
-  <header>
-    {{#if this.loadingHeaderComponent}}
-      <i class="fa fa-circle-o-notch fa-spin"></i>
-    {{else if this.headerComponent}}
-      <this.headerComponent.component @handler={{@handler}} @column={{@column}} @extra={{this.headerComponent.args}} />
-    {{/if}}
+<div id={{this.elementId}} class={{this.computedClasses}} ...attributes>
```

Comment:
Just a question, you did the same in other template, but handle `class` and `attributes` in an independent ways will not bite you if you want to customize the class with the attribute no ?

Reply:
if the `...attributes` contain a `class` attribute, it will be added to the existing classes. the best practice is to keep `...attributes` at the end to avoid overriding it and having uncontrolled behaviour